libgadget/sfr_eff.c

```diff
@@ -621,7 +620,7 @@ starformation(int i, double *localsfr, double * sum_sm)
 
 /* Get the parameters of the basic effective
  * equation of state model for a particle.*/
-static struct sfr_eeqos_data get_sfr_eeqos(int i, double dtime)
+static struct sfr_eeqos_data get_sfr_eeqos(struct particle_data * part, struct sph_particle_data * sph, double dtime)
```

Comment:
struct xxx_particle_data is probably the only place where we have intentionally avoided typedefing the struct; I don't know why; might be to preserve the Gadget-ness of the code.

Reply:
For another cleanup :-)